website/content/api-docs/json-jobs.mdx

```diff
@@ -1055,11 +1055,37 @@ README][ct].
   - `"noop"` - take no action (continue running the task)
   - `"restart"` - restart the task
   - `"signal"` - send a configurable signal to the task
+  - `"script"` - run a script
 
 - `ChangeSignal` - Specifies the signal to send to the task as a string like
   "SIGUSR1" or "SIGINT". This option is required if the `ChangeMode` is
   `signal`.
 
+- `ChangeScriptConfig` - Configures the script triggered on template change.
+  This option is required if the `change_mode` is `script`.
+
+  The `ChangeScriptConfig` object supports the following attributes:
+
+  - `path` - Specifies the full path to a script that is to be executed on
+    template change. This option is required is the `change_mode` is `script`.
+  
+  - `args` - List of arguments that are passed to the script that is to be
+    executed on template change. 
+  
+  - `timeout` - Timeout for script execution specified using a label suffix like
+    "30s" or "1h". This option is required if the `change_mode` is `script`.
+  
+  - `fail_on_error` - If `true`, Nomad will kill the task if the script
+    execution fails. If `false`, script failure will be logged but the task will
+    continue uninterrupted. 
+
+- `ChangeScriptArguments` - Comma-separated list of arguments that are passed to
```

Comment:
Do you still need these `ChangeScriptFoo` blocks?  I don't think so. Also the first one references comma separated lists which you moved away from. I think you can just delete this stuff yeah?  I'm not blockking on this though.